### PR TITLE
Cancel upload

### PR DESCRIPTION
**Pull request by EJohnF**
nothing more than just fix merge conflicts for https://github.com/Vydia/react-native-background-upload/pull/19

### CODE REVIEW

**Comment by TSMMark** [comment]
Thanks! Code looks good to me. @StevePotter? We'll be able to test much quicker if you get a chance please add the functionality into the example app here and submit a PR: https://github.com/Vydia/ReactNativeBackgroundUploadExample

**Comment by EJohnF** [comment]
@TSMMark done: https://github.com/Vydia/ReactNativeBackgroundUploadExample/pull/4

File: android/src/main/java/com/vydia/UploaderModule.java [hunk 1]
```diff
@@ -194,6 +194,7 @@ public void onCancelled(Context context, UploadInfo uploadInfo) {
                 .addFileToUpload(filePath, options.getString("field"));
       }
 
+
```
Comment (Bobryk) [inline android/src/main/java/com/vydia/UploaderModule.java]: no need for new line

**Comment by TSMMark** [comment]
Thanks for your contribution, I'll post here when a new release is live.

**Comment by superandrew213** [comment]
@TSMMark will you be releasing new version soon?

**Comment by StevePotter** [comment]
@superandrew213 releasing today

**Comment by StevePotter** [comment]
@superandrew213 published to npm version 4.2.0.  Enjoy

**Pull request merged**